### DISCUSSION TITLE
24/7 usage

### DISCUSSION THREAD

**Discussion by meinsm** [EslaMx7/ScreenTask#54] [General]
Hello,
if i want to use it 24/7 for a long period of time there is a lot data written to disk, right?
Is it better to store the image in a variable and display that in the webserver?

is it possible to do that?

**Comment by EslaMx7** [comment]
Hello @meinsm 

You are correct, I've started working on such optimization to prevent writing to the disk and serve the image directly from memory, it might take some time to test it properly in different scenarios to avoid common concurrency problems.

**Reply by meinsm** [reply]
Hi, that sounds perfect.
Thank you.
